doc/api/fs.md

```diff
@@ -2693,7 +2748,7 @@ Synchronous rmdir(2). Returns `undefined`.
 Using `fs.rmdirSync()` on a file (not a directory) results in an `ENOENT` error
 on Windows and an `ENOTDIR` error on POSIX.
 
-## fs.stat(path, callback)
+## fs.stat(path[, options], callback)
```

Comment:
When we change a heading, we should grep all-docs-wide to fix all possible cross-links. It seems there are two more with old hashes:

[deprecations.md#L1037](https://github.com/nodejs/node/blame/985d180855d1d57f64ee213ed72fb460328f53f0/doc/api/deprecations.md#L1037)

[fs.md#L4681](https://github.com/nodejs/node/blame/53822f605d3171530ab820d71e8832f473479c27/doc/api/fs.md#L4681)